facade/src/main/java/com/redhat/lightblue/migrator/facade/TimeoutConfiguration.java

```diff
@@ -118,7 +118,17 @@ public long getMS(String methodName, FacadeOperation op, Type type) {
                 log.debug("{} config not found for bean {} using global timeout", type, beanName);
             }
 
-            timeout = defaultTimeoutMS;
+            switch (type) {
+                case timeout: {
+                    timeout = defaultTimeoutMS; break;
+                }
+                case slowwarning: {
+                    timeout = 2 * defaultTimeoutMS; break;
```

Comment:
>> let the constructor handle this?

> Different types of timeouts may have different values. There is only one TimeoutConfiguration instance per facade

I don't understand the need to double the defaultTimeoutMS when it is set by the constructor.

Maybe just some documentation that states that the slowwarning type will just double the defaultTimeoutMS.